### PR TITLE
Update LINDA to use a real process instead of using the scheduler as a

### PR DESCRIPTION
**Pull request by tigercat2000**
ticker

title basically

@Fox-McCloud Requesting a review

For players: This won't have any real effect; Might alleviate some lag problems with LINDA, cannot confirm.

For coders: This gives LINDA a proper scheck() system, so it will actually listen to the process scheduler trying to manage performance.

### CODE REVIEW

File: code/controllers/Processes/air.dm [hunk 1]
```diff
@@ -1,22 +1,121 @@
...
+	var/list/excited_groups = list()
+	var/list/active_turfs = list()
+	var/list/hotspots = list()
+
+	//Special functions lists
+	var/list/turf/simulated/active_super_conductivity = list()
+	var/list/turf/simulated/high_pressure_delta = list()
+
+	var/current_cycle = 0
+	var/failed_ticks = 0
```
Comment (Fox-McCloud) [inline code/controllers/Processes/air.dm]: Is this even necessary anymore? It's currently utilized by the MC (along with air_processing_killed), but I"m not sure if it's really necessary anymore or not. Meh.

**Comment by Fox-McCloud** [comment]
This looks good to me, though I'd like to re-review this tomorrow when I"m not so tired.

File: code/controllers/Processes/air.dm (hunk 1 above)
Comment (tigercat2000) [inline code/controllers/Processes/air.dm]: Nope, basically none of this is actually 'necessary'
But might as well keep it incase it does become useful

File: code/controllers/Processes/air.dm [hunk 2]
```diff
@@ -1,22 +1,121 @@
...
+	plmaster.icon_state = "plasma"
+	plmaster.layer = FLY_LAYER
+	plmaster.mouse_opacity = 0
 
-				air_processing_killed = TRUE
-				air_master.failed_ticks = 0
+	slmaster = new /obj/effect/overlay()
+	slmaster.icon = 'icons/effects/tile_effects.dmi'
+	slmaster.icon_state = "sleeping_agent"
+	slmaster.layer = FLY_LAYER
```
Comment (Fox-McCloud) [inline code/controllers/Processes/air.dm]: missing slmaster.mouse_opacity = 0

**Comment by Fox-McCloud** [comment]
Other than my most recent comment, this looks good to me :+1:

**Pull request merged**